fapolicy_analyzer/ui/strings.py

```diff
@@ -99,7 +99,8 @@
 PROF_ARG_OK = _("Profiler Session arguments are valid.")
 PROF_ARG_EXEC_EMPTY = _("Executable field is empty")
 PROF_ARG_EXEC_DOESNT_EXIST = _(": file does not exist.")
-PROF_ARG_EXEC_NOT_EXECUTABLE = _(": file is not executable")
+PROF_ARG_EXEC_NOT_EXEC = _(": file is not executable")
+PROF_ARG_EXEC_NOT_FOUND = _(": command not found")
 PROF_ARG_USER_DOESNT_EXIST = _(": user does not exist")
 PROF_ARG_PWD_DOESNT_EXIST = _(": working directory does not exist.")
 PROF_ARG_PWD_ISNT_DIR = _(": working directory is not a directory.")
```

Comment:
Bit of a nitpick here, but some of these error messages end in a period and some do not.

Reply:
Made consistent; removed periods.